spring-integration-redis/src/main/resources/org/springframework/integration/redis/config/spring-integration-redis-4.3.xsd

```diff
@@ -506,6 +506,24 @@
 					</xsd:appinfo>
 				</xsd:annotation>
 			</xsd:attribute>
+			<xsd:attribute name="receive-timeout" type="xsd:string" default="1000">
+				<xsd:annotation>
+					<xsd:documentation>
+						Specify the timeout in milliseconds to wait for the result of the
+						'rightPop' operation on Redis queue.
+						Default is 1 second.
+					</xsd:documentation>
+				</xsd:annotation>
+			</xsd:attribute>
+			<xsd:attribute name="recovery-interval" type="xsd:string" default="5000">
```

Comment:
Well, I can add it, indeed. No reason to break rules regarding coverage.
Together with the `null` fix, of course.
If you don't do that on merge...